src/momento/logs.py

```diff
@@ -38,12 +38,10 @@ def add_logging_level(level_name: str, level_num: int, method_name: Optional[str
     if not method_name:
         method_name = level_name.lower()
 
-    if hasattr(logging, level_name):
-        raise AttributeError("{} already defined in logging module".format(level_name))
-    if hasattr(logging, method_name):
-        raise AttributeError("{} already defined in logging module".format(method_name))
-    if hasattr(logging.getLoggerClass(), method_name):
-        raise AttributeError("{} already defined in logger class".format(method_name))
+    if hasattr(logging, level_name) or hasattr(logging, method_name) or hasattr(logging.getLoggerClass(), method_name):
```

Comment:
I suppose someone could make a trace logging level with a different method name than trace and we would be in trouble. Is there anything we can do about that?

Reply:
I think a warning would be appropriate. See [here](https://docs.python.org/3/library/warnings.html)

Reply:
Good idea